File: PHP/Film.php
<?php
class Film {
    private string $id;
    private string $judul;
    private string $genre;
    private int $durasi;
    private string $gambar;

    public function __construct(string $id, string $judul, string $genre, int $durasi, string $gambar) {
        $this->id = $id;
        $this->judul = $judul;
        $this->genre = $genre;
        $this->durasi = $durasi;
        $this->gambar = $gambar;
    }

    public function getId(): string { return $this->id; }
    public function getJudul(): string { return $this->judul; }
    public function getGenre(): string { return $this->genre; }
    public function getDurasi(): int { return $this->durasi; }
    public function getGambar(): string { return $this->gambar; }

    public function setJudul(string $judul): void { $this->judul = $judul; }
    public function setGenre(string $genre): void { $this->genre = $genre; }
    public function setDurasi(int $durasi): void { $this->durasi = $durasi; }
    public function setGambar(string $gambar): void { $this->gambar = $gambar; }
}
?>

File: Python/Film.py
class Film:
    def __init__(self, id_film: str, judul: str, genre: str, durasi: int, gambar: str):
        self._id = str(id_film)
        self._judul = str(judul)
        self._genre = str(genre)
        self._durasi = int(durasi)
        self._gambar = str(gambar)

    # Getters
    def getId(self) -> str:
        return self._id
    def getJudul(self) -> str:
        return self._judul
    def getGenre(self) -> str:
        return self._genre
    def getDurasi(self) -> int:
        return self._durasi
    def getGambar(self) -> str:
        return self._gambar

    # Setters
    def setJudul(self, judul: str) -> None:
        self._judul = str(judul)
    def setGenre(self, genre: str) -> None:
        self._genre = str(genre)
    def setDurasi(self, durasi: int) -> None:
        self._durasi = int(durasi)
    def setGambar(self, gambar: str) -> None:
        self._gambar = str(gambar)